user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+----
+git clone https://github.com/boostorg/boost.git -b boost-{latest_version} {release_basename} --depth 1 <1>
+cd {release_basename}
+git submodule update --depth 1 --init --recursive <2>
+----
+
+<.> Clone the Boost super-project
+<.> Initialize all boost modules
+--
+====
+
+==== Installing Individual Boost Modules
+
+The process above downloads all boost libraries, which is the usual process in a local installation.
+In projects where only a subset of the Boost libraries is required, the following procedure can be used:
+
+[source,bash,subs="attributes+"]
+.Downloading boost:unordered[] and internal dependencies
+----
+git clone https://github.com/boostorg/boost.git -b boost-{latest_version} {release_basename} --depth 1 <1>
+git submodule update --depth 1 -q --init tools/boostdep <2>
+git submodule update --depth 1 -q --init libs/unordered <3>
+python tools/boostdep/depinst/depinst.py -X test -g "--depth 1" unordered <4>
+----
+
+<1> Download the Boost super-project, which contains references to all Boost libraries
+<2> Initialize boost:boostdep[] used to determine internal boost dependencies
+<3> Initialize boost:unordered[].
+Repeat this step for any other modules you need.
+<4> Initialize any dependencies of boost:unordered[].
+Repeat this step for any other modules you need.
+
+This procedure requires `git` and `python`.
+The following steps in this document apply to both methods of installing Boost.
+The only difference is what libraries will be available.
+
+=== Bootstrap
+
+If you followed the steps above, your current working directory should be `{release_basename}`.
+Otherwise, open up a command line terminal, and navigate to the boost folder you extracted.
+
+[tabs,sync-group-id=build]
+====
+Boost.Build::
++
+--
+The Boost libraries are built using a custom build app called boost:build[].
+This app itself is built by running the bootstrap script.
+
+This is the usual procedure to configure, build and install Boost with boost:build[]:
+
+Decide where you are going to install the libraries to, most locations are OK, something like *Documents/Boost-libraries* is fine.
+If your location has a space in the name, you will need to surround the path in quotes in the commands that follow.
+
+Windows:
+
+[source]
+----
+bootstrap.bat
+b2
+b2 install --prefix=C:\Boost
+----
+
+If you get an error during the build process such as the following, you probably do not have an up to date and full installation of pass:[C++] on your machine.
+Go back to <<Prerequisites>> and install or update your version of Visual Studio.
+
+image::cpp-runtime-error.png[]
+
+Unix variants:
+
+[source]
+----
+./bootstrap.sh
+./b2
+./b2 install --prefix=/usr/local
+----
+
+You can adjust the installation `prefix` to your preference in this command.
+
+NOTE:: We break the build and install into separate steps, in order to get better diagnostics when building the libraries, and to suppress too many messages when installing the libraries.
```

Comment:
We're mentioning breaking the build into two steps becaue of `-d0` but there's no `-d0` in any of the commands